src/metabase/api/pulse.clj

```diff
@@ -127,12 +131,13 @@
 
 (defendpoint POST "/test"
   "Test send an unsaved pulse."
-  [:as {{:keys [name cards channels] :as body} :body}]
-  {name     su/NonBlankString
-   cards    (su/non-empty [su/Map])
-   channels (su/non-empty [su/Map])}
+  [:as {{:keys [name cards channels skip_if_empty] :as body} :body}]
+  {name          su/NonBlankString
+   cards         (su/non-empty [su/Map])
+   channels      (su/non-empty [su/Map])
+   skip_if_empty s/Bool}
   (check-card-read-permissions cards)
   (p/send-pulse! body)
-  {:ok true})
+  (assoc body :ok true))
```

Comment:
it helped me see what was being passed, and as it is a test function I didn't mind returning more stuff but it can be removed.
Anyway I'm assoc'ing a `:skip` parameter that tells if the pulse is skipped so the ui can display a msg, I think at least this param is necessary